src/EditorFeatures/Core.Wpf/Suggestions/FixAll/FixAllGetFixesService.cs

```diff
@@ -157,12 +157,23 @@ private static async Task<CodeAction> GetFixAllCodeActionAsync(FixAllContext fix
                 }),
                 cancellationToken))
             {
-                var previewService = workspace.Services.GetService<IPreviewDialogService>();
                 var glyph = languageOpt == null
                     ? Glyph.Assembly
                     : languageOpt == LanguageNames.CSharp
                         ? Glyph.CSharpProject
                         : Glyph.BasicProject;
+#if COCOA
+
+                var previewService = workspace.Services.GetService<IPreviewDialogService>();
+
+                // Until IPreviewDialogService is implemented, just execute all changes without user ability to pick and choose
+                if (previewService == null)
```

Comment:
I could make the Cocoa branch the branch used on both sides, but was worried we might then lose the comment/intent and also the assert that on the VS for Windows side it _should_ work.

Reply:
I'd leave this as-is since we do want the preview dialog on Windows.

Reply:
@KirillOsenkov The cocoa side was already nicely doing light-up, so that would work.